server/Episememe.Application.Tests/FeatureTests/MediaRevision/WhenRevisingMedia.cs

```diff
@@ -23,60 +21,80 @@ public class WhenRevisingMedia : IDisposable
 
         public WhenRevisingMedia()
         {
-            (_contextMock, _connection) = GetInMemoryDatabaseContext();
+            (_contextMock, _connection) = InMemoryDatabaseFactory.CreateSqliteDbContext();
         }
 
         [Fact]
-        public void GivenMediaTagsAreUpdated_ThenExpectedMediaTagsAreInDatabase ()
+        public void GivenMediaTagsAreUpdated_ThenExpectedMediaTagsAreInDatabase()
         {
-            var Tags = new List<string> 
-            {"sword", "shield", "minimini"};
+            _contextMock.MediaInstances.Add(CreateExampleDatabaseInstance());
+            _contextMock.SaveChanges();
+            var tags = new List<string>
+            {
+                "sword", "shield", "minimini"
+            };
 
-            var command = UpdateTagsCommand.Create("k8wetest", Tags);
+            var command = UpdateTagsCommand.Create("k8wetest", tags, string.Empty);
             var handler = new UpdateTagsCommandHandler(_contextMock);
             handler.Handle(command, CancellationToken.None).Wait();
 
-            _contextMock.MediaInstances.Single().MediaTags.Select(t => t.Tag.Name).Should().BeEquivalentTo(Tags);
+            _contextMock.MediaInstances.Single().MediaTags.Select(t => t.Tag.Name).Should().BeEquivalentTo(tags);
         }
 
         [Fact]
-        public void GivenNonexistentFileID_ThenArgumentExcpetionThrown()
+        public void GivenNonexistentFileID_ThenAggregateExceptionIsThrown()
         {
-            var Tags = new List<string> 
-            {"sword", "shield", "minimini"};
+            _contextMock.MediaInstances.Add(CreateExampleDatabaseInstance());
+            _contextMock.SaveChanges();
+            var tags = new List<string>
+            {
+                "sword", "shield", "minimini"
+            };
 
-            var command = UpdateTagsCommand.Create("xdxdxdxd", Tags);
+            var command = UpdateTagsCommand.Create("xdxdxdxd", tags, string.Empty);
             var handler = new UpdateTagsCommandHandler(_contextMock);
-            
+
             Action act = () => handler.Handle(command, CancellationToken.None).Wait();
 
-            act.Should().Throw<ArgumentException>();
+            act.Should().Throw<AggregateException>();
```

Comment:
That seems closely coupled to the implementation. I think a more general type should be used, unless we specifically want this particular exception.
E.g.: if I test a file system access component, I do want to check if FileDoesNotExistException is thrown. However, handlers can contain various pieces of code, so specific exceptions may not be a good idea